docs/source/install_rocm.rst

```diff
@@ -40,7 +40,7 @@ Install CuPy from Source
 ------------------------
 
 It is recommended to use wheels whenever possible.
-However, if wheels cannot meet your requirements (e.g., you are running non-Linux environment or want to use different version sets of CUDA / cuDNN / NCCL that are not included in any CuPy wheels), you have to build CuPy from source.
+However, there is currently no wheels for ROCm environment, so that you have to build it from source.
```

Comment:
Just for minor grammar changes:
```suggestion
However, there is currently no wheels for the ROCm environment, so you have to build it from source.
```

Reply:
Thanks for the suggestion! I applied it.